### ISSUE TITLE
iOS/iPadOS notification prompt for Web Push Notifications

### ISSUE THREAD

**Issue by tarunkumar519** [sct/overseerr#3524]
### Description

iOS now supports web push notifications starting with iOS 16.4. In iOS settings > Safari> Advanced > Enable Push API and websites now may send notifications. Seems to be working with websites that enabled this (ex: twitter refer : https://www.xda-developers.com/how-enable-safari-notifications-iphone/)

This is working perfectly on iOS 16.4 and newer but notification prompt only occours after some changes are made to any settings in overseerr and inbuilt overseerr notifications arise.

### Desired Behavior

When a website like Twitter is added to homescreen on iPhone/iPad twitter asks to allow push notifications using a button. If we click it a regular iOS notification prompt asks if to allow notifications or scheduled or dont allow.

Overseerr should have this button in settings under webpush so users can click on it to activate notification prompt.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by OwsleyJr** [comment]
Already an open PR for this. #3421

**Comment by tarunkumar519** [comment]
I agree with that pull request. But as per overseerr wiki docs, iOS notifications does not work. Since it does now, an update on that and also how to enable it in iOS would be good.

**Comment by KevinEdry** [comment]
Just popped in to check if there is an open issue for PWA notifications and there is! Waiting for this to be merged :)

**Comment by mylogon341** [comment]
It would be great to get some movement on this. Is there anyone that can look at this @tarunkumar519 ?

**Comment by tarunkumar519** [comment]
Having a button to prompt would be great. Currently there is no button, but overseerr asks to allow notifications at random time after making app PWA on iOS,iPadOS. I just open the app a few times and this prompt pops up. I have done this for my users. This prompt should be easy, i will look into this or maybe someone can help
